rollup.config.js

```diff
@@ -20,6 +20,8 @@ const ts_plugin = is_publish
 
 const external = id => id.startsWith('svelte/');
 
+fs.writeFileSync(`./compiler.d.ts`, `export * from './types/compiler/index';`);
```

Comment:
If this is a static file that will always contain this same thing, why create it as part of the build and not just check it into the repo as we're currently doing?

Reply:
That was just for consistency since the other `.d.ts` and `package.json`s are all static too.

At first I thought it was an error to be included until I realized that it was a static file, but all of them are :P

Reply:
Hmmm okay. I guess I'm kind of undecided about this. They may all be static files, but the `.d.ts` and `package.json` files would need to be generated for each directory, and if we ever added a new runtime export path, we the current rollup config would automatically generate new ones. I dunno. I really don't have a big opinion about this either way I guess.